cmd/entrypoint/waiter_test.go

```diff
@@ -38,7 +39,7 @@ func TestRealWaiterWaitMissingFile(t *testing.T) {
 	rw := realWaiter{}
 	doneCh := make(chan struct{})
 	go func() {
-		err := rw.setWaitPollingInterval(testWaitPollingInterval).Wait(tmp.Name(), false, false)
+		err := rw.setWaitPollingInterval(testWaitPollingInterval).Wait(context.Background(), tmp.Name(), false, false)
```

Comment:
For readability, maybe `ctx := context.Background()` up at the top of this test, and reuse it throughout.

Reply:
Thank you for the review. Fixed.